### DISCUSSION TITLE
how to change just artist name and not track name?

### DISCUSSION THREAD

**Discussion by militarylcheryl** [Taiko2k/Tauon#1099] [General]
im customing my theme and i want to be able to change the artist name color, n not the track name
![image](https://github.com/Taiko2k/TauonMusicBox/assets/162048020/55c3e65c-cfd4-4892-9d37-3d1da83866d2)
in the theme file changing "title info" changes both n just wondering if is possible, tyty ^^

**Comment by Taiko2k** [comment]
Yeah this isn't currently changeable, both of those lines are mapped to that same colour.

**Reply by militarylcheryl** [reply]
:( oh okay, well thank you!
